orttraining/orttraining/python/training/ortmodule/experimental/hierarchical_ortmodule/_hierarchical_ortmodule.py

```diff
@@ -130,34 +115,40 @@ def recursive_wrap(module):
 
         # Try exporter on all module-input pairs. If a module can be exported with
         # all its recorded inputs, then it's exporable.
-        check_exportable(self._module)
+        check_exportable(self._wrapped_module)
 
         # A naive way of determining if ORT can run nn.Module
         def is_supported(module):
             # This is what Yuan called GetCapability().
```

Comment:
```suggestion
```
Either remove it or rephrase to prevent names

Reply:
Removed. My bad.